test/libsolidity/GasMeter.cpp

```diff
@@ -47,7 +47,9 @@ class GasMeterTestFramework: public SolidityExecutionFramework
 	GasMeterTestFramework() { }
 	void compile(string const& _sourceCode)
 	{
-		m_compiler.setSource("pragma solidity >= 0.0;" + _sourceCode);
+		m_compiler.reset(false);
+		m_compiler.addSource("", "pragma solidity >= 0.0;" + _sourceCode);
+		/// NOTE: compiles without optimisations
```

Comment:
This was the case before too - is this a desired property in the `GasMeterTests`? Should it run with the optimiser on when requested (second pass of the tests)?